src/main/java/com/zepben/evolve/streaming/get/NetworkConsumerClient.kt

```diff
@@ -7,24 +7,21 @@
  */
 package com.zepben.evolve.streaming.get
 
-import com.zepben.evolve.cim.iec61970.base.core.ConductingEquipment
 import com.zepben.evolve.cim.iec61970.base.core.Feeder
 import com.zepben.evolve.cim.iec61970.base.core.IdentifiedObject
-import com.zepben.evolve.cim.iec61970.base.wires.AcLineSegment
-import com.zepben.evolve.cim.iec61970.base.wires.Conductor
 import com.zepben.evolve.services.common.BaseService
-import com.zepben.evolve.services.common.Resolvers
 import com.zepben.evolve.services.common.extensions.typeNameAndMRID
 import com.zepben.evolve.services.common.translator.mRID
 import com.zepben.evolve.services.network.NetworkService
 import com.zepben.evolve.services.network.translator.NetworkProtoToCim
 import com.zepben.evolve.services.network.translator.mRID
 import com.zepben.evolve.streaming.get.hierarchy.*
+import com.zepben.evolve.streaming.get.hierarchy.NetworkHierarchyFeeder
```

Comment:
Are these imports required? I would imagine they are covered by the `.*` above.

Reply:
i'm not sure what that's about. funnily enough if i remove the * import, NetworkHierarchy doesn't work, and if i remove the other imports, the * doesn't work for them. Both the IDE and maven complain... seems like a kotlin bug. i wouldn't be surprised if its been fixed and we just need to update the kotlin version (although i don't want to do that in this PR)